frontend/src/metabase/modes/components/drill/DashboardClickDrill.tsx

```diff
@@ -57,20 +65,20 @@ export default ({ question, clicked }) => {
       behavior = {
         ignoreSiteUrl: true,
         url: () =>
-          renderLinkURLForClick(clickBehavior.linkTemplate || "", data),
+          renderLinkURLForClick(clickBehavior.linkTemplate || "", data as any),
```

Comment:
Could we align the types here? They seem really close, guess there's just some notation weirdness